README.md

```diff
@@ -219,6 +219,30 @@ There are 2 global options available. Both will apply to all functions that foll
  * `@default_metric_options`: Metric options to use unless overridden with `@metric_options`. Defaults to [].
  * `@use_histogram`: Send results using histograms instead of gauges. For use with Datadog's DogStatD. Defaults to false.
 
+## Starting your own ExStatsD Server
+
+There may be use cases where you need several `ExStatsD` instances. You may have several `namespaces` you need to send stats to, or maybe you have multiple statsd services you need to publish to.
+
+To achieve this you can manually start a `ExStatsD` instance.
+
+```elixir
+  {:ok, _first_pid} = ExStatsD.start_link([name: :first, namespace: "first.namespace"])
+
+  {:ok, _second_pid} = ExStatsD.start_link([name: :second, namespace: "second.namespace"])
+
+  ExStatsD.increment("the.thing", name: :first) # nil
+  ExStatsD.increment("another.thing", name: :second) # nil
+```
+
+Of course, it's your own responsibility to supervise these servers.
```

Comment:
Just for new people, can you add an example of what this might look like as workers? Basically just

```
[
  worker(ExStatsD, [[name: :second, namespace: "second.namespace"]]),
  ... other_children
]
```